### DISCUSSION TITLE
Pregunta Examen

### DISCUSSION THREAD

**Discussion by JavierQuisphe** [mmasias/24-25-PRG1#580] [Q&A] [answered]
Buenas tarde profesor una consulta, en la parte 2 del examen no se especifica la cantidad de caramelos que recibe Carmen como base, unicamente se especifica que recibe caramelos extra en el piso 4 y 5

**Comment by mmasias** [comment, marked as answer]
@JavierQuisphe ¡buen punto! :) Supongamos de 1 a 3 caramelos.

**Reply by mmasias** [reply]
> @JavierQuisphe ¡buen punto! :) Supongamos de 1 a 3 caramelos.

que, de hecho, mantiene las condiciones originales (que se cambiaron para los otros dos chicos): _en cada casa, cuando dan caramelos, dan entre 1 y 3 caramelos_

**Reply by JavierQuisphe** [reply]
va, gracias. Tengo otra duda y es que, yo agregué una variable para que cambien de bolsa cuando bajen del edificio para que puedan ir a recoger dulces a los demas edificios a pesar de que en la tarea no dijera eso, no se si está correcto, pero me hizo sentido ya que sino no nos haria agregar 9 edificios mas si el problema iba a terminar maximo en el 2do

**Reply by mmasias** [reply]
> va, gracias. Tengo otra duda y es que, yo agregué una variable para que cambien de bolsa cuando bajen del edificio para que puedan ir a recoger dulces a los demas edificios a pesar de que en la tarea no dijera eso, no se si está correcto, pero me hizo sentido ya que sino no nos haria agregar 9 edificios mas si el problema iba a terminar maximo en el 2do

😄 Dale un vistazo a la reflexión en https://github.com/mmasias/24-25-PRG1/discussions/581#discussioncomment-11157263

**Reply by JavierQuisphe** [reply]
listo, supongo que en ese caso ya seria mas a nuestra creatividad.
